### ISSUE TITLE
Overlay CPU Became 0 on wattage and temperature.

### ISSUE THREAD

**Issue by TangKeke98** [Valkirie/HandheldCompanion#983]
**Device manufacturer**
- [ ] AYANEO
- [ ] AYN
- [ ] AOKZOE
- [ ] ASUS
- [ ] GPD
- [ ] ONEXPLAYER
- [*] VALVE

**Device model**
Steam Deck LCD

**Handheld Companion Version**
0.20.4.1

**Describe the bug**
CPU Wattage and Temp became 0 even I disabled overlay and turn on back still not working, even restart Handheld Companion also didn't work, I restarted device to make it working, sometimes after device restart still show 0w and 0C, reinstall HC to make working again. Load are working as intended.

**Comment by MSeys** [comment]
@0SkillAllLuck sounds like ADLX is not always providing the info? At least on Steam Deck.

**Comment by CasperH2O** [comment]
@TangKeke98 thanks for dropping in. Did you put your device in sleep/hibernate by any chance?

**Comment by 0SkillAllLuck** [comment]
CPU data is comming from LHM not ADLX.
Weird that usage works but not temp or power

**Comment by TangKeke98** [comment]
> @TangKeke98 thanks for dropping in. Did you put your device in sleep/hibernate by any chance?

Yes but sometimes, hibernate?? nah I not enabled it that feature in my windows. After gaming session I always shutdown my Windows after gaming session.

**Comment by BriaN890501** [comment]
Similar issue on Steam Deck LCD but also with GPU usage, wattage, VRAM
The cpu usage also seems abnormaly high?
![9e2b0625-5c0c-4dcd-8fc6-eff43fc6dba0](https://github.com/Valkirie/HandheldCompanion/assets/45000802/9bea15f2-8ed1-4527-a643-e03e637190dc)
![f3b6ca04-3c98-484d-b10e-04f9def04dbb](https://github.com/Valkirie/HandheldCompanion/assets/45000802/df416b8a-466b-4ce1-8041-8ecff772c547)

It seemed to have happened after recent version moved away from HWinfo

**Comment by huangting** [comment]
Hello, have you resolved this issue? 
Sometimes the temperature is still displayed as 0c in 0.21.2.4, Steam Deck LCD.
And 0c can cause the fan stoping if using software fan control.
Sometimes rebooting deck can solve the problem, but sometimes it doesn't.
Thank you

**Comment by Lukasviedenz** [comment]
Can confirm this Issue is still present anyone found a fix ?
